bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:50 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:50 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:50 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:50 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:50 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:50 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:50 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:50 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:51 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:51 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:51 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:51 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:51 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:51 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:52 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:52 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:55 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:52:55 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:53:03 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:53:03 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:53:23 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:53:23 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:53:43 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:53:43 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:54:03 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:54:03 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:54:24 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:54:24 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:54:44 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:54:44 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:55:04 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:55:04 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:55:24 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:55:24 PM
Docker Error:  js-oj-container Docker Error : {"msg":"on docker error","error":null,"stderr":"Unable to find image 'node:16.17.0-bullseye-slim' locally\n16.17.0-bullseye-slim: Pulling from library/node\n31b3f1ad4ce1: Pulling fs layer\nc973db49e5a0: Pulling fs layer\n69b0052faebf: Pulling fs layer\nc1e0dc661a4e: Pulling fs layer\n38b58f98de7a: Pulling fs layer\nc1e0dc661a4e: Waiting\n38b58f98de7a: Waiting\nc973db49e5a0: Verifying Checksum\nc973db49e5a0: Download complete\nc1e0dc661a4e: Verifying Checksum\nc1e0dc661a4e: Download complete\n38b58f98de7a: Verifying Checksum\n38b58f98de7a: Download complete\n31b3f1ad4ce1: Verifying Checksum\n31b3f1ad4ce1: Download complete\n69b0052faebf: Verifying Checksum\n69b0052faebf: Download complete\n31b3f1ad4ce1: Pull complete\nc973db49e5a0: Pull complete\n69b0052faebf: Pull complete\nc1e0dc661a4e: Pull complete\n38b58f98de7a: Pull complete\nDigest: sha256:59812c19504546fc66b0b26722bf0754ee48b74f9abc5ed9c3f251fc45d86099\nStatus: Downloaded newer image for node:16.17.0-bullseye-slim\n"}
9/15/2024, 1:55:43 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:55:44 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:55:44 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:56:04 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:56:04 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:56:24 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:56:24 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:56:44 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 9/15/2024, 1:56:44 PM
Docker Error:  gcc-oj-container Docker Error : {"msg":"on docker error","error":{"code":125,"killed":false,"signal":null,"cmd":"docker run -i -d --rm --mount type=bind,src=\"E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeFiles\",dst=/codeFiles --name gcc-oj-container --label oj=oj gcc:latest"},"stderr":"docker: Error response from daemon: Conflict. The container name \"/gcc-oj-container\" is already in use by container \"7d8d3cb27b5086c8065e5b241f712b8deb8cdb809139d8cfb1d6c28d93b6a0c9\". You have to remove (or rename) that container to be able to reuse that name.\nSee 'docker run --help'.\n"}
9/15/2024, 1:57:31 PM
Error: Illegal arguments: string, undefined
    at _async (E:\gith\Online-Judge-Project-master\server\node_modules\bcryptjs\dist\bcrypt.js:286:46)
    at E:\gith\Online-Judge-Project-master\server\node_modules\bcryptjs\dist\bcrypt.js:307:17
    at new Promise (<anonymous>)
    at Object.bcrypt.compare (E:\gith\Online-Judge-Project-master\server\node_modules\bcryptjs\dist\bcrypt.js:306:20)
    at loginValidatorHelper (E:\gith\Online-Judge-Project-master\server\middlewares.js:15:42)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async loginValidator (E:\gith\Online-Judge-Project-master\server\middlewares.js:36:9) 9/15/2024, 2:00:55 PM
Error: Illegal arguments: string, undefined
    at _async (E:\gith\Online-Judge-Project-master\server\node_modules\bcryptjs\dist\bcrypt.js:286:46)
    at E:\gith\Online-Judge-Project-master\server\node_modules\bcryptjs\dist\bcrypt.js:307:17
    at new Promise (<anonymous>)
    at Object.bcrypt.compare (E:\gith\Online-Judge-Project-master\server\node_modules\bcryptjs\dist\bcrypt.js:306:20)
    at loginValidatorHelper (E:\gith\Online-Judge-Project-master\server\middlewares.js:15:42)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async loginValidator (E:\gith\Online-Judge-Project-master\server\middlewares.js:36:9) 9/15/2024, 2:00:59 PM
Error without msg in bull.process Error: Cannot find module './testcases/undefined'
Require stack:
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js
- E:\gith\Online-Judge-Project-master\server\controllers\explore.js
- E:\gith\Online-Judge-Project-master\server\routes\explore.js
- E:\gith\Online-Judge-Project-master\server\index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at execCodeAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js:94:31)
    at processAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:32:26)
    at Queue.<anonymous> (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:52:48)
    at handlers.<computed> (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:701:42)
    at Queue.processJob (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:1162:22)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeExecutor_dockerv.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\queryQueue.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\controllers\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\routes\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\index.js'
  ]
} 9/15/2024, 2:06:42 PM
Error without msg in bull.process Error: Cannot find module './testcases/undefined'
Require stack:
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js
- E:\gith\Online-Judge-Project-master\server\controllers\explore.js
- E:\gith\Online-Judge-Project-master\server\routes\explore.js
- E:\gith\Online-Judge-Project-master\server\index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at execCodeAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js:94:31)
    at processAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:32:26)
    at Queue.<anonymous> (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:52:48)
    at handlers.<computed> (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:701:42)
    at Queue.processJob (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:1162:22)
    at runMicrotasks (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeExecutor_dockerv.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\queryQueue.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\controllers\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\routes\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\index.js'
  ]
} 9/15/2024, 2:15:01 PM
Error without msg in bull.process Error: Cannot find module './testcases/function binarySearch(arr, target) { let left = 0; let right = arr.length - 1; while (left <= right) { const mid = Math.floor((left + right) / 2); if (arr[mid] === target) return mid; else if (arr[mid] < target) left = mid + 1; else right = mid - 1; } return -1; }'
Require stack:
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js
- E:\gith\Online-Judge-Project-master\server\controllers\explore.js
- E:\gith\Online-Judge-Project-master\server\routes\explore.js
- E:\gith\Online-Judge-Project-master\server\index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at execCodeAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js:94:31)
    at processAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:32:26)
    at Queue.<anonymous> (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:52:48)
    at handlers.<computed> (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:701:42)
    at Queue.processJob (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:1162:22)
    at runMicrotasks (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeExecutor_dockerv.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\queryQueue.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\controllers\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\routes\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\index.js'
  ]
} 9/15/2024, 3:39:06 PM
Error without msg in bull.process Error: Cannot find module './testcases/function binarySearch(arr, target) { let left = 0; let right = arr.length - 1; while (left <= right) { const mid = Math.floor((left + right) / 2); if (arr[mid] === target) return mid; else if (arr[mid] < target) left = mid + 1; else right = mid - 1; } return -1; }'
Require stack:
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js
- E:\gith\Online-Judge-Project-master\server\controllers\explore.js
- E:\gith\Online-Judge-Project-master\server\routes\explore.js
- E:\gith\Online-Judge-Project-master\server\index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at execCodeAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js:94:31)
    at processAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:32:26)
    at Queue.<anonymous> (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:52:48)
    at handlers.<computed> (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:701:42)
    at Queue.processJob (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:1162:22)
    at runMicrotasks (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeExecutor_dockerv.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\queryQueue.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\controllers\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\routes\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\index.js'
  ]
} 9/15/2024, 3:40:05 PM
Error without msg in bull.process Error: Cannot find module './testcases/function binarySearch(arr, target) { let left = 0; let right = arr.length - 1; while (left <= right) { const mid = Math.floor((left + right) / 2); if (arr[mid] === target) return mid; else if (arr[mid] < target) left = mid + 1; else right = mid - 1; } return -1; }'
Require stack:
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js
- E:\gith\Online-Judge-Project-master\server\controllers\explore.js
- E:\gith\Online-Judge-Project-master\server\routes\explore.js
- E:\gith\Online-Judge-Project-master\server\index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at execCodeAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js:94:31)
    at processAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:32:26)
    at Queue.<anonymous> (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:52:48)
    at handlers.<computed> (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:701:42)
    at Queue.processJob (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:1162:22)
    at runMicrotasks (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeExecutor_dockerv.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\queryQueue.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\controllers\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\routes\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\index.js'
  ]
} 9/15/2024, 3:43:34 PM
Error while inserting log into the database: MongooseServerSelectionError: getaddrinfo ENOTFOUND test-shard-00-00.qwazf.mongodb.net
    at Function.Model.$wrapCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:5007:32)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:510:27
    at promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:10:12)
    at Mongoose._promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\index.js:1220:10)
    at model.Model.save (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:509:35)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3138:16
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3168:7
    at Array.forEach (<anonymous>)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3167:15
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:32:5 {
  reason: TopologyDescription {
    type: 'ReplicaSetNoPrimary',
    servers: Map(3) {
      'test-shard-00-00.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-01.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-02.qwazf.mongodb.net:27017' => [ServerDescription]
    },
    stale: false,
    compatible: true,
    heartbeatFrequencyMS: 10000,
    localThresholdMS: 15,
    setName: 'atlas-uego3w-shard-0',
    maxSetVersion: 1,
    maxElectionId: new ObjectId("7fffffff0000000000000003"),
    commonWireVersion: 21,
    logicalSessionTimeoutMinutes: undefined
  },
  code: undefined
}
MongooseServerSelectionError: getaddrinfo ENOTFOUND test-shard-00-00.qwazf.mongodb.net
    at Function.Model.$wrapCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:5007:32)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:510:27
    at promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:10:12)
    at Mongoose._promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\index.js:1220:10)
    at model.Model.save (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:509:35)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3138:16
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3168:7
    at Array.forEach (<anonymous>)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3167:15
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:32:5 {
  reason: TopologyDescription {
    type: 'ReplicaSetNoPrimary',
    servers: Map(3) {
      'test-shard-00-00.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-01.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-02.qwazf.mongodb.net:27017' => [ServerDescription]
    },
    stale: false,
    compatible: true,
    heartbeatFrequencyMS: 10000,
    localThresholdMS: 15,
    setName: 'atlas-uego3w-shard-0',
    maxSetVersion: 1,
    maxElectionId: new ObjectId("7fffffff0000000000000003"),
    commonWireVersion: 21,
    logicalSessionTimeoutMinutes: undefined
  },
  code: undefined
} 9/15/2024, 3:47:07 PM
Error while inserting log into the database: MongooseServerSelectionError: getaddrinfo ENOTFOUND test-shard-00-00.qwazf.mongodb.net
    at Function.Model.$wrapCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:5007:32)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:510:27
    at promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:10:12)
    at Mongoose._promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\index.js:1220:10)
    at model.Model.save (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:509:35)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3138:16
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3168:7
    at Array.forEach (<anonymous>)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3167:15
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:32:5 {
  reason: TopologyDescription {
    type: 'ReplicaSetNoPrimary',
    servers: Map(3) {
      'test-shard-00-00.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-01.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-02.qwazf.mongodb.net:27017' => [ServerDescription]
    },
    stale: false,
    compatible: true,
    heartbeatFrequencyMS: 10000,
    localThresholdMS: 15,
    setName: 'atlas-uego3w-shard-0',
    maxSetVersion: 1,
    maxElectionId: new ObjectId("7fffffff0000000000000003"),
    commonWireVersion: 21,
    logicalSessionTimeoutMinutes: undefined
  },
  code: undefined
}
MongooseServerSelectionError: getaddrinfo ENOTFOUND test-shard-00-00.qwazf.mongodb.net
    at Function.Model.$wrapCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:5007:32)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:510:27
    at promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:10:12)
    at Mongoose._promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\index.js:1220:10)
    at model.Model.save (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:509:35)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3138:16
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3168:7
    at Array.forEach (<anonymous>)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3167:15
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:32:5 {
  reason: TopologyDescription {
    type: 'ReplicaSetNoPrimary',
    servers: Map(3) {
      'test-shard-00-00.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-01.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-02.qwazf.mongodb.net:27017' => [ServerDescription]
    },
    stale: false,
    compatible: true,
    heartbeatFrequencyMS: 10000,
    localThresholdMS: 15,
    setName: 'atlas-uego3w-shard-0',
    maxSetVersion: 1,
    maxElectionId: new ObjectId("7fffffff0000000000000003"),
    commonWireVersion: 21,
    logicalSessionTimeoutMinutes: undefined
  },
  code: undefined
} 9/15/2024, 3:47:25 PM
Error while inserting log into the database: MongooseServerSelectionError: getaddrinfo ENOTFOUND test-shard-00-00.qwazf.mongodb.net
    at Function.Model.$wrapCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:5007:32)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:510:27
    at promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:10:12)
    at Mongoose._promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\index.js:1220:10)
    at model.Model.save (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:509:35)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3138:16
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3168:7
    at Array.forEach (<anonymous>)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3167:15
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:32:5 {
  reason: TopologyDescription {
    type: 'ReplicaSetNoPrimary',
    servers: Map(3) {
      'test-shard-00-00.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-01.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-02.qwazf.mongodb.net:27017' => [ServerDescription]
    },
    stale: false,
    compatible: true,
    heartbeatFrequencyMS: 10000,
    localThresholdMS: 15,
    setName: 'atlas-uego3w-shard-0',
    maxSetVersion: 1,
    maxElectionId: new ObjectId("7fffffff0000000000000003"),
    commonWireVersion: 21,
    logicalSessionTimeoutMinutes: undefined
  },
  code: undefined
}
Error while inserting log into the database: MongooseServerSelectionError: connection timed out
    at Function.Model.$wrapCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:5007:32)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:510:27
    at promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:10:12)
    at Mongoose._promiseOrCallback (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\index.js:1220:10)
    at model.Model.save (E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:509:35)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3138:16
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3168:7
    at Array.forEach (<anonymous>)
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\model.js:3167:15
    at E:\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\helpers\promiseOrCallback.js:32:5 {
  reason: TopologyDescription {
    type: 'ReplicaSetNoPrimary',
    servers: Map(3) {
      'test-shard-00-00.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-01.qwazf.mongodb.net:27017' => [ServerDescription],
      'test-shard-00-02.qwazf.mongodb.net:27017' => [ServerDescription]
    },
    stale: false,
    compatible: true,
    heartbeatFrequencyMS: 10000,
    localThresholdMS: 15,
    setName: 'atlas-uego3w-shard-0',
    maxSetVersion: 1,
    maxElectionId: new ObjectId("7fffffff0000000000000003"),
    commonWireVersion: 21,
    logicalSessionTimeoutMinutes: undefined
  },
  code: undefined
}
TypeError: Cannot read properties of null (reading 'name')
    at loggedInController (E:\gith\Online-Judge-Project-master\server\controllers\user.js:81:24)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 9/15/2024, 7:12:31 PM
Error without msg in bull.process Error: Cannot find module './testcases/function binarySearch(arr, target) { let left = 0; let right = arr.length - 1; while (left <= right) { const mid = Math.floor((left + right) / 2); if (arr[mid] === target) return mid; else if (arr[mid] < target) left = mid + 1; else right = mid - 1; } return -1; }'
Require stack:
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js
- E:\gith\Online-Judge-Project-master\server\controllers\explore.js
- E:\gith\Online-Judge-Project-master\server\routes\explore.js
- E:\gith\Online-Judge-Project-master\server\index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at execCodeAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js:94:31)
    at processAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:32:26)
    at Queue.<anonymous> (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:52:48)
    at handlers.<computed> (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:701:42)
    at Queue.processJob (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:1162:22)
    at runMicrotasks (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeExecutor_dockerv.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\queryQueue.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\controllers\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\routes\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\index.js'
  ]
} 9/15/2024, 7:18:54 PM
Error without msg in bull.process Error: Cannot find module './testcases/undefined'
Require stack:
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js
- E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js
- E:\gith\Online-Judge-Project-master\server\controllers\explore.js
- E:\gith\Online-Judge-Project-master\server\routes\explore.js
- E:\gith\Online-Judge-Project-master\server\index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at execCodeAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js:94:31)
    at processAgainstTestcases (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:32:26)
    at Queue.<anonymous> (E:\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:52:48)
    at handlers.<computed> (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:701:42)
    at Queue.processJob (E:\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:1162:22)
    at runMicrotasks (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeExecutor_dockerv.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\queryQueue.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\controllers\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\routes\\explore.js',
    'E:\\gith\\Online-Judge-Project-master\\server\\index.js'
  ]
} 9/15/2024, 7:25:44 PM
Docker Error:  java-oj-container Docker Error : {"msg":"on docker error","error":{"code":125,"killed":false,"signal":null,"cmd":"docker run -i -d --rm --mount type=bind,src=\"E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeFiles\",dst=/codeFiles --name java-oj-container --label oj=oj openjdk:20-slim"},"stderr":"docker: Error response from daemon: Conflict. The container name \"/java-oj-container\" is already in use by container \"8c82b8f19a9fdfb16e3a6589227042e442a694c4e84a7a638b58fff5e0c19cc9\". You have to remove (or rename) that container to be able to reuse that name.\nSee 'docker run --help'.\n"}
9/15/2024, 7:37:45 PM
Docker Error:  py-oj-container Docker Error : {"msg":"on docker error","error":{"code":125,"killed":false,"signal":null,"cmd":"docker run -i -d --rm --mount type=bind,src=\"E:\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeFiles\",dst=/codeFiles --name py-oj-container --label oj=oj python:3.10-slim"},"stderr":"docker: Error response from daemon: Conflict. The container name \"/py-oj-container\" is already in use by container \"3dc30f9b99a07bec478f95b6619ef8e70542ef95e37ddec9df5652dab1b34820\". You have to remove (or rename) that container to be able to reuse that name.\nSee 'docker run --help'.\n"}
9/15/2024, 8:49:22 PM
Error without msg in bull.process Error: Could not find or load main class Solution
Caused by: java.lang.ClassNotFoundException: Solution
 9/15/2024, 10:24:15 PM
Error without msg in bull.process Error: Could not find or load main class Solution
Caused by: java.lang.ClassNotFoundException: Solution
 9/15/2024, 10:24:23 PM
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
[31mOh no MONGOOSE Error !!! 2/21/2025, 1:56:15 PM[39m
Error: querySrv ETIMEOUT _mongodb._tcp.test.qwazf.mongodb.net
    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/callback_resolver:45:19) {
  errno: undefined,
  code: 'ETIMEOUT',
  syscall: 'querySrv',
  hostname: '_mongodb._tcp.test.qwazf.mongodb.net'
}
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
MongooseError: Operation `users.findOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7) 2/21/2025, 2:08:44 PM
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
MongooseError: Operation `users.findOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7) 2/21/2025, 2:09:24 PM
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
[31mOh no MONGOOSE Error !!! 2/21/2025, 2:14:53 PM[39m
Error: querySrv ETIMEOUT _mongodb._tcp.test2.edwtg.mongodb.net
    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/callback_resolver:45:19) {
  errno: undefined,
  code: 'ETIMEOUT',
  syscall: 'querySrv',
  hostname: '_mongodb._tcp.test2.edwtg.mongodb.net'
}
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
MongooseError: Operation `users.findOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7) 2/21/2025, 2:15:25 PM
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error while inserting log into the database: MongooseError: Operation `logs.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:151:23)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
Error without msg in bull.process Error: Cannot find module './testcases/[1, 2, 3, 4, 5], target = 5'
Require stack:
- E:\BTech Projects\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js
- E:\BTech Projects\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js
- E:\BTech Projects\gith\Online-Judge-Project-master\server\controllers\explore.js
- E:\BTech Projects\gith\Online-Judge-Project-master\server\routes\explore.js
- E:\BTech Projects\gith\Online-Judge-Project-master\server\index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at execCodeAgainstTestcases (E:\BTech Projects\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js:94:31)
    at processAgainstTestcases (E:\BTech Projects\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:32:26)
    at Queue.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:52:48)
    at handlers.<computed> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:701:42)
    at Queue.processJob (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:1162:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeExecutor_dockerv.js',
    'E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\queryQueue.js',
    'E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\controllers\\explore.js',
    'E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\routes\\explore.js',
    'E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\index.js'
  ]
} 2/21/2025, 2:26:51 PM
Error without msg in bull.process Error: Cannot find module './testcases/[1, 2, 3, 4, 5], target = 5'
Require stack:
- E:\BTech Projects\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js
- E:\BTech Projects\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js
- E:\BTech Projects\gith\Online-Judge-Project-master\server\controllers\explore.js
- E:\BTech Projects\gith\Online-Judge-Project-master\server\routes\explore.js
- E:\BTech Projects\gith\Online-Judge-Project-master\server\index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at execCodeAgainstTestcases (E:\BTech Projects\gith\Online-Judge-Project-master\server\CodeExecuter\codeExecutor_dockerv.js:94:31)
    at processAgainstTestcases (E:\BTech Projects\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:32:26)
    at Queue.<anonymous> (E:\BTech Projects\gith\Online-Judge-Project-master\server\CodeExecuter\queryQueue.js:52:48)
    at handlers.<computed> (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:701:42)
    at Queue.processJob (E:\BTech Projects\gith\Online-Judge-Project-master\server\node_modules\bull\lib\queue.js:1162:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeExecutor_dockerv.js',
    'E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\queryQueue.js',
    'E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\controllers\\explore.js',
    'E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\routes\\explore.js',
    'E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\index.js'
  ]
} 2/21/2025, 2:33:58 PM
[31mDocker Error:  java-oj-container Docker Error : {"msg":"on docker error","error":{"code":125,"killed":false,"signal":null,"cmd":"docker run -i -d --rm --mount type=bind,src=\"E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeFiles\",dst=/codeFiles --name java-oj-container --label oj=oj openjdk:20-slim"},"stderr":"docker: Error response from daemon: Conflict. The container name \"/java-oj-container\" is already in use by container \"f3dfaa03224cbb61f6907fa2b2e24931b5f2bf2cb48e330ca71d64b7a7f2955c\". You have to remove (or rename) that container to be able to reuse that name.\n\nRun 'docker run --help' for more information\n"}[39m
[31m4/18/2025, 5:17:50 PM[39m
[31mDocker Error:  gcc-oj-container Docker Error : {"msg":"on docker error","error":{"code":125,"killed":false,"signal":null,"cmd":"docker run -i -d --rm --mount type=bind,src=\"E:\\BTech Projects\\gith\\Online-Judge-Project-master\\server\\CodeExecuter\\codeFiles\",dst=/codeFiles --name gcc-oj-container --label oj=oj gcc:latest"},"stderr":"Unable to find image 'gcc:latest' locally\nlatest: Pulling from library/gcc\na0fcfad35b18: Pulling fs layer\ne4dd9c002dd2: Pulling fs layer\n8b77cd7514d6: Pulling fs layer\nea589c0b318a: Pulling fs layer\nb617a119f8a2: Pulling fs layer\ndocker: failed to extract layer sha256:46f98f0a19e17ebae7acf046d39afde89be6097e7dd32cd81321384dbc9aca71: open /var/lib/desktop-containerd/daemon/io.containerd.snapshotter.v1.overlayfs/snapshots/107/fs/usr/local/include/c++/14.2.0/ext/pb_ds/detail/pairing_heap_/debug_fn_imps.hpp: read-only file system\n\nRun 'docker run --help' for more information\n"}[39m
[31m4/18/2025, 5:19:19 PM[39m
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:49 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:49 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:49 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:49 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:49 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:49 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:49 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:49 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:49 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:49 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:49 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:50 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:50 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:50 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:51 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:51 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:54 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 9:59:54 PM
Docker Error:  js-oj-container Docker Error : {"msg":"on docker error","error":{"code":127,"killed":false,"signal":null,"cmd":"docker run -i -d --rm --mount type=bind,src=\"D:\\Chrome Downloads\\Project-main\\Project-main\\server\\CodeExecuter\\codeFiles\",dst=/codeFiles --name js-oj-container --label oj=oj node:16.17.0-bullseye-slim"},"stderr":"docker: error during connect: Head \"http://%2F%2F.%2Fpipe%2FdockerDesktopLinuxEngine/_ping\": open //./pipe/dockerDesktopLinuxEngine: The system cannot find the file specified.\n\nRun 'docker run --help' for more information\n"}
4/18/2025, 9:59:55 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:00:02 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:00:02 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:00:22 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:00:22 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
Docker Error:  py-oj-container Docker Error : {"msg":"on docker error","error":{"code":127,"killed":false,"signal":null,"cmd":"docker run -i -d --rm --mount type=bind,src=\"D:\\Chrome Downloads\\Project-main\\Project-main\\server\\CodeExecuter\\codeFiles\",dst=/codeFiles --name py-oj-container --label oj=oj python:3.10-slim"},"stderr":"docker: error during connect: Head \"http://%2F%2F.%2Fpipe%2FdockerDesktopLinuxEngine/_ping\": open //./pipe/dockerDesktopLinuxEngine: The system cannot find the file specified.\n\nRun 'docker run --help' for more information\n"}
4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:13 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:14 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:14 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:15 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:15 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:18 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:18 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:04:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:05:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:05:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:05:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:05:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:05:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:05:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:06:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:06:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:06:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:06:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:06:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:06:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:07:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:07:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:07:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:07:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:07:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:07:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:08:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:08:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:08:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:08:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:08:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:08:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:09:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:09:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:09:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:09:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:09:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:09:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:10:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:10:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:10:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:10:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:10:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:10:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:11:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:11:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:11:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:11:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:11:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:11:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:12:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:12:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:12:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:12:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:12:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:12:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:13:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:13:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:13:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:13:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:13:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:13:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:14:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:14:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:14:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:14:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:14:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:14:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:15:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:15:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:15:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:15:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:15:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:15:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:16:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:16:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:16:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:16:26 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:16:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:16:46 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:17:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/18/2025, 10:17:06 PM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:14 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:15 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:15 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:15 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:15 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:15 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:16 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:16 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:16 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:16 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:16 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:16 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:16 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:17 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:17 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:18 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:20 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:21 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:28 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:29 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:48 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:29:49 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:07 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:08 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:08 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:09 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:09 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:12 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:12 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:20 AM
bull/redis error Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1634:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379
} 4/19/2025, 7:30:20 AM
